code/modules/mob/living/carbon/human/species_types/eggpeople.dm

```diff
@@ -0,0 +1,42 @@
+// This is the dumbest fucking thing I have ever programmed t. Altoids
+#define EGG_MAXBRUTEMOD 1.5
+#define EGG_ALPHA 0.00018
+#define EGG_BETA (-0.027)
+/datum/species/egg
+	// Egghead humanoids composed of eggs and stale memes
+	name = "Eggperson"
+	id = "egg"
+	say_mod = "blurbles"
+	fixed_mut_color = "FFE7C9"
+	offset_features = list(OFFSET_UNIFORM = list(0,0), OFFSET_ID = list(0,0), OFFSET_GLOVES = list(0,0), OFFSET_GLASSES = list(0,0), OFFSET_EARS = list(0,2), OFFSET_SHOES = list(0,0), OFFSET_S_STORE = list(0,0), OFFSET_FACEMASK = list(0,0), OFFSET_HEAD = list(0,2), OFFSET_FACE = list(0,0), OFFSET_BELT = list(0,0), OFFSET_BACK = list(0,0), OFFSET_SUIT = list(0,0), OFFSET_NECK = list(0,0))
+	species_traits = list(EYECOLOR,LIPS, MUTCOLORS)
+	inherent_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	brutemod = EGG_MAXBRUTEMOD // not the toughest egg in the dozen (handled by the #defines above)
```

Comment:
Replace the define with the actual value here and in the apply damage just initial(brutemod), allows for more easily making subtypes answer its a tad More readable although slower at runtime I suppose

Reply:
Eh, moving the number into a define just a dozen lines up isn't too much of a problem for readability, especially since, as you said, it'd slightly affect performance to do otherwise.

Reply:
I marked this as resolved because i noticed how dumb it was